composer.json

```diff
@@ -60,8 +60,7 @@
         "knplabs/knp-time-bundle": "^1.10",
         "lcobucci/jwt": "^3.4 || ^4.0",
         "league/commonmark": "^1.6",
-        "league/uri-components": "^1.8",
-        "league/uri-schemes": "^1.2",
+        "league/uri-components": "^2.4",
```

Comment:
We are also using this in other places, so if we decide to go without, we have to adjust those as well.